src/node/blockstorage.h

```diff
@@ -70,6 +71,30 @@ struct PruneLockInfo {
     int height_first{std::numeric_limits<int>::max()}; //! Height of earliest block that should be kept and not pruned
 };
 
+enum class BlockfileType {
+    // For the purposes of blockfile fragmentation, treat background IBD chainstates
+    // as normal.
```

Comment:
re: https://github.com/bitcoin/bitcoin/pull/27596#discussion_r1311988064

In commit "blockstorage: segment normal/assumedvalid blockfiles" (3ff26bc73374b53550dc2dd17dd3e1c58258fc30)

> I've just removed the comment because I don't think it adds much anymore.

Comment seems to be here still (and would be good to remove)